lib/sneeze-board.web-view.tsx

```diff
@@ -161,7 +162,7 @@ globalThis.webViewComponent = function SneezeBoard() {
       return;
     }
 
-    dataProvider?.setAchYou('NEWUSER', {
+    dataProvider?.setUser('NEWUSER', {
       userId: 'thisShouldBeUniqueButIsNotRightNow',
```

Comment:
If you'd like, you could use `newGuid()` from one of the utility files! You could also leave it off here and set it in the backend if you wanted to have a bit more control over the data. Just a thought; don't need to worry about that now if you don't want to!